lib/node_modules/@stdlib/math/base/ops/cneg/docs/types/index.d.ts

```diff
@@ -45,6 +45,7 @@ import { Complex128 } from '@stdlib/types/object';
 * var im = imag( out );
 * // returns -5.5
 *
+* @example
```

Comment:
When split into separate examples, we need to also include the import statements on L32-34. Examples in JSDoc comments should be independently executable code. Currently, apart from the first example, the subsequent `@example` blocks cannot execute because they'll have undefined variable errors (`Complex128`, `real`, and `imag`). The only variable which does **not** need to be explicitly imported/required is the main export `cneg` as this is magically added to the local example execution context during linting.

Reply:
Looks like you added the imports for the subsequent files, just not this file.